configure.ac

```diff
@@ -180,6 +180,9 @@ AC_ARG_ENABLE(forced_server_mode,
 AC_ARG_ENABLE(debug,
               [  --enable-debug      Enable debugging code.],, enable_debug="no")
 
+AC_ARG_ENABLE(force_gnu99,
+              [  --enable-force-gnu99      Enforce C99 with gnu extensions.],, force_gnu99="no")
```

Comment:
Can we turn this on by default?

Reply:
I wanted to do it as opt-in as c99 is something anybody should really force. As you may miss some compiler optimisation in this way - even if your codebase follows c99 (I do not say it is a fact, just a feeling). This is not really a *good* enough reason.

Why would you run this on by default ?

Reply:
I just wanted to avoid the possibility to write some code locally, that would not compile in CI: just for catching the errors easier. But I can make an "alias" for this option in my development environment, so not really important request. Marking as resolved.